### PR TITLE
Fixing small bug in program

### PR DESCRIPTION
**Pull request by zmhassan**
# Issue
In production sometimes the message length can be larger than 200 chars which will break the system. So I've increase the limit to 2000 char for log messages.

### CODE REVIEW

**Comment by zmhassan** [comment]
@durandom This is a hot fix. Please merge this. I've already deployed this to PSI

File: Pipfile [hunk 1]
```diff
@@ -31,6 +31,7 @@ pyyaml = "*"
 Flask-SQLAlchemy = "*"
 PyMySQL = "*"
 numba = "*"
+cryptography = "*"
```
Comment (durandom) [inline Pipfile]: @zmhassan how is this needed in this PR?

File: anomaly_detector/anomaly_detector.py [hunk 2]
```diff
@@ -70,9 +70,9 @@ def __init__(self, cfg: Configuration):
 
     def _load_data(self, time_span, max_entries, false_positives=None):
         """Loading data from storage into pandas dataframe for processing."""
-        data, raw = self.storage.retrieve(time_range=time_span,
-                                          number_of_entries=max_entries,
-                                          false_data=false_positives)
```
Comment (durandom) [inline anomaly_detector/anomaly_detector.py]: should'nt we use named arguments as seen in https://github.com/AICoE/log-anomaly-detector/search?q=retrieve&unscoped_q=retrieve

Comment (zmhassan) [reply anomaly_detector/anomaly_detector.py]: The named arguments is broken. It's failing to work.

File: Pipfile (hunk 1 above)
Comment (zmhassan) [reply Pipfile]: When I run locally on mac with mysql and fact_store running it fails to work. Unless I install this package.

**Comment by zmhassan** [comment]
[[KR9] Prometheus Metrics Improvements](https://trello.com/c/6R73EuHw/309-kr9-prometheus-metrics-improvements)

**Comment by durandom** [comment]
> 1. the title doesnt say anything
> 2. this PR does more than one thing.
> 
> please change only the 255 to 2000 setting and not other stuff

@zmhassan please fix those 2 points in this PR and open other PRs for the other stuff

**Comment by zmhassan** [comment]
Since you mentioned that you have standards to PR's and I can only have 1 fix per PR. I'm going to close this PR and break it up into individual PR's that address each problem.

**Pull request closed without merging**